deploy.sh

```diff
@@ -70,6 +70,11 @@ do
 
   DEPLOY_DOMAIN=https://${DEPLOY_SUBDOMAIN}-${REPO_NAME}-${REPO_OWNER}.surge.sh
 
+  # Rebuild with NODE_ENV=production for test target only
+  if [ "$NODE_ENV" == "test" ]; then
+    NODE_ENV=production npm run build
```

Comment:
As @stefaniacardenas has mentioned, this would mean we lose the alert about the SMS link when using the the Surge links for manual testing or acceptance. 
Maybe what we really need is a solution like [this](https://stackoverflow.com/a/53368078) where in our case a `TEST_ENV` is set to differentiate the testing environments.